### DISCUSSION TITLE
generic build command

### DISCUSSION THREAD

**Discussion by jacknicklenson** [premake/premake-core#1743] [Q&A] [answered]
Hi, how can i build a project after generating build files eg. in cmake after you write let's say : ```cmake ..``` we can do this : 
```cmake --build .``` to build without thinking about build system, it take cares of build system and build it. How we can achive this in premake ?

**Comment by nickclark2016** [comment, marked as answer]
This is not currently supported in Premake, as it just emits build system files.

**Reply by jacknicklenson** [reply]
so is there any plan to add ?

**Reply by nickclark2016** [reply]
I am not aware of a plan, but I'd have to defer to @starkos or @samsinsane to confirm.

**Reply by starkos** [reply]
I don't believe anyone is working on a feature like this currently, but it would be a nice addition!
